python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -66,7 +66,7 @@ def prepare(self, project: "DbtProject") -> None:
             DbtCliResource(project_dir=os.fspath(project.project_dir))
             .cli(
                 self._generate_cli_args,
-                target_path=project.target_dir,
+                target_path=project.manifest_path.parent,
```

Comment:
Why this change? This should respect `target_dir` still right? 

The expectation in the package data flow is that we run prepare in to the source folder first and then sync it to package data second.

Reply:
We should be able to revert this once we set `project_dir=current_project_dir`